docs/source/installation.rst

```diff
@@ -207,6 +207,22 @@ Some users have experienced issues when installing NumPy, particularly version 1
     Another option under Windows is to follow the instructions for `Unix-like OS`_, using the Windows Subsystem for Linux (WSL).
     For installing WSL, follow the instructions `here <https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-2---check-requirements-for-running-wsl-2>`_.
 
+Some quick tips on virtual environments
+---------------------------------
```

Comment:
header should be at the same level as "Setting up a development environment", and underline should be same length as header text